data/forcing/forcings-engine/config_aorc.yml

```diff
@@ -0,0 +1,64 @@
+time_step_seconds: 3600
+initial_time: 0
+NWM_VERSION: 4.0
+NWM_CONFIG: "AORC"
+InputForcings: [12] 
+InputForcingDirectories: ['']
+InputForcingTypes: ["NETCDF4"]
+InputMandatory: [1]
+OutputFrequency: 60
+SubOutputHour: 0
+SubOutFreq: 0
+ScratchDir: "data/forcing/forcings-engine/scratch"
+Output: 1
+compressOutput: 0
+floatOutput: 0
+AnAFlag: 0
+LookBack : -9999
+RefcstBDateProc: "202301170000"
+RefcstEDateProc: "202301170100"
```

Comment:
These are the time configurations that conflict with `time_start` and `duration` arguments higher up, right?

Reply:
Yeah, correct.